sonar/config.py

```diff
@@ -279,6 +280,39 @@ def _(x):
         update_permission_factory_imp=allow_all,
         delete_permission_factory_imp=allow_all,
         list_permission_factory_imp=allow_all
+    ),
+    'user': dict(
+        pid_type='user',
+        pid_minter='user_id',
+        pid_fetcher='user_id',
+        default_endpoint_prefix=True,
+        record_class=UserRecord,
+        search_class=UserSearch,
+        indexer_class=RecordIndexer,
+        search_index='users',
+        search_type=None,
+        record_serializers={
+            'application/json': ('sonar.modules.users.serializers'
+                                 ':json_v1_response'),
+        },
+        search_serializers={
+            'application/json': ('sonar.modules.users.serializers'
+                                 ':json_v1_search'),
+        },
+        record_loaders={
+            'application/json': ('sonar.modules.users.loaders'
+                                 ':json_v1'),
+        },
+        list_route='/users/',
+        item_route='/users/<pid(user):pid_value>',
+        default_media_type='application/json',
+        max_result_window=10000,
+        error_handlers=dict(),
+        create_permission_factory_imp=allow_all,
```

Comment:
really? Anybody can create and update a user?

Reply:
yes, you're right, but can I add this checks in the PR #71, because logic for permissions is added in that PR.